src/main/java/org/glovo/controller/OrderController.java

```diff
@@ -0,0 +1,64 @@
+package org.glovo.controller;
+
+import lombok.AllArgsConstructor;
+import org.glovo.model.Order;
+import org.glovo.model.Product;
+import org.glovo.service.OrderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+
+    private final OrderService orderService;
+
+    @GetMapping()
+    public List<Order> getAllOrders() {
```

Comment:
it is orderController so you don't need to add the word Order to the method, this comment is related to all methods